src/dlgRoomExits.cpp

```diff
@@ -758,6 +897,54 @@ void dlgRoomExits::save()
+        }
+
+        // This is the toolTip text for the room ID number column (and the
+        // status icon)
+        const QString roomIdToolTipText{generateToolTip(pExitToRoom->name, exitAreaName, outOfAreaExit, pExitToRoom->getWeight())};
+        pSpecialExit->setIcon(ExitsTreeWidget::colIndex_exitStatus, showIconOnExitStatus ? (outOfAreaExit ? mIcon_otherAreaExit : mIcon_inAreaExit) : QIcon());
+        pSpecialExit->setToolTip(ExitsTreeWidget::colIndex_exitRoomId, roomIdToolTipText);
+        pSpecialExit->setToolTip(ExitsTreeWidget::colIndex_exitStatus, roomIdToolTipText);
+
+    } else if (pSpecialExit->text(ExitsTreeWidget::colIndex_exitRoomId).toInt() > 0) {
+        // A number but not valid
+        pSpecialExit->setIcon(ExitsTreeWidget::colIndex_exitStatus, showIconOnExitStatus ? mIcon_invalidExit : QIcon());
+        pSpecialExit->setToolTip(ExitsTreeWidget::colIndex_exitRoomId, singleParagraph.arg(tr("Entered number is invalid, set the number of the room that this special exit leads to; "
+                                                                                              "if left like this, this exit will be deleted when <tt>save</tt> is clicked.")));
+    } else if (pSpecialExit->text(ExitsTreeWidget::colIndex_exitRoomId).isEmpty() || pSpecialExit->text(ExitsTreeWidget::colIndex_exitRoomId) == mSpecialExitRoomIdPlaceholder) {
+        // Nothing
+        pSpecialExit->setIcon(ExitsTreeWidget::colIndex_exitStatus, QIcon());
+        pSpecialExit->setToolTip(ExitsTreeWidget::colIndex_exitRoomId, singleParagraph.arg(tr("A positive room ID of the room that this special exit leads to is expected here; "
+                                                                                              "if left like this, this exit will be deleted when <tt>save</tt> is clicked.")));
+    } else {
+        // Something else that isn't a positive number
+        pSpecialExit->setIcon(ExitsTreeWidget::colIndex_exitStatus, showIconOnExitStatus ? mIcon_invalidExit : QIcon());
+        pSpecialExit->setToolTip(ExitsTreeWidget::colIndex_exitRoomId, singleParagraph.arg(tr("A positive room ID of the room that this special exit leads to is expected here; "
+                                                                                              "if left like this, this exit will be deleted when <tt>save</tt> is clicked.")));
+    }
+
+    if (pSpecialExit->text(ExitsTreeWidget::colIndex_command) == mSpecialExitCommandPlaceholder) {
+        pSpecialExit->setToolTip(ExitsTreeWidget::colIndex_command, singleParagraph.arg(tr("No command or Lua script entered, if left like this, this exit will be deleted when <tt>save</tt> is clicked.")));
+    } else {
+        pSpecialExit->setToolTip(ExitsTreeWidget::colIndex_command, singleParagraph.arg(tr("(Lua scripts for some mapper packages {those using the <tt>mudlet-mapper/<tt>, i.e. I.R.E. Muds and StickMUD} need to be prefixed with \"script:\").")));
```

Comment:
I don't think it's wise naming the games here, because the list will change and this string will age unneccessarily
```suggestion
        pSpecialExit->setToolTip(ExitsTreeWidget::colIndex_command, singleParagraph.arg(tr("(Lua scripts for mapper packages using the <tt>mudlet-mapper/<tt> need to be prefixed with \"script:\").")));
```
Then again, I never even knew of this prerequisite, or how to describe it better maybe.

Reply:
I only found out when I looked to see where it was used - it isn't anything in the C++ core code but something in the Lua stuff that makes up that package and the original text always niggled me, it being `"(Lua scripts need to be prefixed with \"script:\")"` when I knew that wasn't the whole story...

Reply:
OK but I was wondering, why is that required then?
Looking at the mudlet-mapper scripts, they actually have it coded:
```lua
  if string.starts(cmd, "script:") then
    cmd = string.gsub(cmd, "script:", "")
    loadstring(cmd)()
```
So now I am puzzled as to why Mudlet would hint at using a game-specific feature at all.
That is not very common, and not done for other features of the same script either.
To me it seems now, this is a slippery slope and maybe should actually be removed.

Reply:
Let's remember that this has not been a source of confusion or a problem for anyone, I suggest not treating it as one until it actually manifests itself.